File: inclusive-edu.py
from flask import Flask, render_template, send_from_directory

app = Flask(__name__)

# Change this to your actual image folder
IMAGE_FOLDER = r"C:\Users\Jan\Downloads\MyStuff\Academics 6\Software Development\inclusive-edu"
app.config["IMAGE_FOLDER"] = IMAGE_FOLDER

@app.route("/")
def splash():
    return render_template("splash.html")

@app.route("/dashboard")
def dashboard():
    return render_template("dashboard.html")  # Make sure dashboard.html is inside /templates

@app.route("/local-image/<filename>")
def get_local_image(filename):
    return send_from_directory(app.config["IMAGE_FOLDER"], filename)

if __name__ == "__main__":
    app.run(debug=True)

File: inclusive-edu.js
const express = require("express");
const path = require("path");

const app = express();
const PORT = 3000; // You can change this if needed

// Set the image folder
const IMAGE_FOLDER = path.join(__dirname, "public");

// Serve static files (HTML, CSS, JS)
app.use(express.static(path.join(__dirname, "public")));

// Route for the splash screen
app.get("/", (req, res) => {
    res.sendFile(path.join(__dirname, "public", "splash.html"));
});

// Route for the dashboard page
app.get("/dashboard", (req, res) => {
    res.sendFile(path.join(__dirname, "public", "dashboard.html"));
});

// Route for serving local images
app.get("/local-image/:filename", (req, res) => {
    res.sendFile(path.join(IMAGE_FOLDER, req.params.filename));
});

// Start the server
app.listen(PORT, () => {
    console.log(`Server running at http://localhost:${PORT}`);
});
